### PR TITLE
Update site_info.json to add TCCON sites info

### PR DESCRIPTION
**Pull request by alexdanjou**
Addition of TCCON site details

### CODE REVIEW

**Review by rt17603 — COMMENTED** [review]
I have a question around this - up until now we've generally included 3-letter site codes for each of our in-situ measurement sites. Was there a reason this included the 2-letter site code (e.g. was this specified somewhere else)? Or did we want to try and stick to the 3-letter codes?

**Comment by rt17603** [comment]
> The reason is that they are referred to with a 2-letter site code in the TCCON data. But I agree that it is not really good given the amount of other sites that we have. I can put a 3-letter code, no pb on this

Yes, that makes sense - 3-letter codes look good though.

**Review by rt17603 — COMMENTED** [review]
Looks good but a few suggested updates:
 - For these specific cases, could we include a `tccon_name` variable so we still have a record of the original name this has in TCCON?
 - Would we be able to include the new sites alphabetically? Hopefully it doesn't make it less practical to not have the TCCON sites grouped, but in the rest of the file the sites are alphabetical so its easy to locate them.

**Comment by alexdanjou** [comment]
Localisations seems to be completely wrong, I will double check the script to create them

**Comment by alexdanjou** [comment]
Should be good now. Some sites have a changing latitude/longitude (change of a few cents of degrees). Put them in an array style but not sure this is the right things to do.
Also, ordering the keys and formatting with an indent of 2 before storing changed the display of the other sites. Is it ok?

**Comment by rt17603** [comment]
Can I just check - has is been reformatted because you ran black (or something else) over it?

**Comment by alexdanjou** [comment]
No, it's because I converted my dict to json with `json.dumps(d, indent=2, sort_keys=True)`

**Comment by alexdanjou** [comment]
Are we good to merge or should I format this otherwise?

**Comment by rt17603** [comment]
> Are we good to merge or should I format this differently?

I am a little concerned about the reformatting across the whole file - just in case anyone else has been using a derivative of this file they'd like to be able to merge back in at some point. Would be it be possible to ensure the updates only impact the added lines or would that be difficult to do?

**Comment by alexdanjou** [comment]
I've done as much as I could to keep the old formattimg. But the identation was not homogeneous and some sites where not stored at their place (in terms of allphabetical order). Reordering just the sites and not the keys inside (i.e. "height", "latitude",..) would have been too much of a pain, so I there is some changes there too.

I think we should go with this as it is. Tell me if you're not ok @rt17603

File: openghg_defs/data/site_info.json [hunk 1]
```diff
@@ -58,9 +58,9 @@
       "height": ["27m"],
       "height_name": ["30magl"],
       "height_station_masl": 20.0,
-      "latitude": 69.6200,
```
Comment (alexdanjou) [inline openghg_defs/data/site_info.json:61]: need the 00 at the end

File: openghg_defs/data/site_info.json [hunk 2]
```diff
@@ -58,9 +58,9 @@
...
+      "latitude": 69.62,
       "long_name": "Ambarchik, Russia",
-      "longitude": 162.3000
```
Comment (alexdanjou) [inline openghg_defs/data/site_info.json:63]: need the 4-digit precision

File: openghg_defs/data/site_info.json [hunk 3]
```diff
@@ -195,7 +195,7 @@
       "height": ["50m"],
       "height_name": ["50magl"],
       "height_station_masl": 110,
-      "latitude": 54.7050,
```
Comment (alexdanjou) [inline openghg_defs/data/site_info.json:198]: need the 4 digit precision

File: openghg_defs/data/site_info.json [hunk 4]
```diff
@@ -3129,31 +3095,31 @@
       "height_station_masl": 64,
       "latitude": 52.51882,
       "long_name": "Tacolneston Tower, UK",
-      "longitude": 1.13870
```
Comment (alexdanjou) [inline openghg_defs/data/site_info.json:3132]: need the 5-digit precision

**Comment by rt17603** [comment]
By the way, I'm not saying this was done perfectly but there was originally a logic to the order of the inlets, with the first generally being the default used. So I think we may need to a bit cautious with reordering the heights lists even if it seems logical to order with largest inlet first.

**Comment by alexdanjou** [comment]
The heights are not reordered as they are not keys to the json files. So that is kept. But the network order is changed some times which might change the first inlet (if all networks have not the same first inlet).
Anyhow I'm planning to delete this PR and go for yours

**Pull request closed without merging**